docs/docs/vulnerability/examples/others.md

```diff
@@ -84,7 +84,7 @@ This flag is available with the following targets.
 - Root filesystem (`trivy rootfs`)
 
 ```
-$ trivy image --exit-code 1 --exit-on-eosl testbeds/ubuntu:10.04
+$ trivy image --exit-on-eol 1 testbeds/ubuntu:10.04
```

Comment:
can we change example for this?

This can be confusing. Example has no vulnerabilities and users can think that Trivy doesn't find vulnerabilities with this flag.
Also after table there is error message:

```
➜  trivy git:(fix/exit-on-eosl) ✗ ./trivy -d image --exit-on-eol 1 alpine:3.12
...
2023-03-01T15:17:38.033+0600    WARN    This OS version is no longer supported by the distribution: alpine 3.12.12
2023-03-01T15:17:38.033+0600    WARN    The vulnerability detection may be insufficient because security updates are not provided

alpine:3.12 (alpine 3.12.12)

Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 1)
...
2023-03-01T15:17:38.034+0600    ERROR   Detected EOL OS: alpine 3.12.12

```

Reply:
Nice feedback! Fixed in https://github.com/aquasecurity/trivy/pull/3716/commits/bababa86ec3a992822c837736313bac0e35abc0f